static/app/views/starfish/views/webServiceView/starfishView.tsx

```diff
@@ -27,68 +29,122 @@ type BasePerformanceViewProps = {
 export function StarfishView(props: BasePerformanceViewProps) {
   const {organization, eventView} = props;
 
+  function renderModuleBreakdownChart() {
+    return (
+      <EventsRequest
+        query={eventView.query}
+        includePrevious={false}
+        partial
+        interval="1h"
+        includeTransformedData
+        limit={1}
+        environment={eventView.environment}
+        project={eventView.project}
+        period={eventView.statsPeriod}
+        referrer="starfish-homepage-span-breakdown"
+        start={eventView.start}
+        end={eventView.end}
+        organization={organization}
+        yAxis={[
+          'p95(spans.db)',
+          'p95(spans.http)',
+          'p95(spans.browser)',
+          'p95(spans.resource)',
+          'p95(spans.ui)',
+        ]}
+        queryExtras={{
+          dataset: 'metrics',
+        }}
+      >
+        {data => {
+          return (
+            <Chart
+              statsPeriod={eventView.statsPeriod}
+              height={180}
+              data={data.results as Series[]}
+              start={eventView.start as string}
+              end={eventView.end as string}
+              loading={data.loading}
+              utc={false}
+              grid={{
+                left: '0',
+                right: '0',
+                top: '16px',
+                bottom: '8px',
+              }}
+              disableMultiAxis
+              definedAxisTicks={4}
+              stacked
+              log
+              chartColors={CHART_PALETTE[5]}
+            />
+          );
+        }}
+      </EventsRequest>
+    );
+  }
+
+  function renderFailureRateChart() {
+    const query = new MutableSearch(['event.type:transaction']);
+
+    return (
+      <EventsRequest
+        query={query.formatString()}
+        includePrevious={false}
+        partial
+        interval="1h"
+        includeTransformedData
+        limit={1}
+        environment={eventView.environment}
+        project={eventView.project}
+        period={eventView.statsPeriod}
+        referrer="starfish-homepage-failure-rate"
+        start={eventView.start}
+        end={eventView.end}
+        organization={organization}
+        yAxis="equation|count_if(http.status_code,greaterOrEquals,500)/(count_if(http.status_code,equals,200)+count_if(http.status_code,greaterOrEquals,500))"
```

Comment:
I think we should include all 200 response [codes](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status#successful_responses) in the denominator since they are all successful responses?